app/views/green_flag/admin/features/index.html.erb

```diff
@@ -6,8 +6,21 @@
 <div class="row">
   <% @features.each do |feature| %>
     <div class="col-md-6">
-      <h4><%= link_to feature.code, admin_feature_path(feature) %></h4>
-      <p><%= feature.description %></p>
+      <div class="well">
+        <h4>
+          <%= link_to feature.code, admin_feature_path(feature) %>
+          <% if feature.fully_enabled? %><span class="label label-success">100%</span><% end %>
+          <% if feature.fully_disabled? %><span class="label label-danger">0%</span><% end %>
+        </h4>
+        
+        <div class="help-block">
+          <div>Created at <strong><%= feature.created_at.strftime("%d %b %Y") %></strong></div>
+          <div>Enabled for <strong><%= feature.enabled_count %> people</strong></div>
+          <div>Disabled for <strong><%= feature.disabled_count %> people</strong></div>
```

Comment:
The `enabled_count` and `disabled_count` methods are slow in production right now, due to the big feature_decisions table.  I have a plan to switch the indexes on that table, which might make this fast enough for the index page.  Do you want to do that in this ticket, or leave it for later?